hedging/tests/hedging.rs

```diff
@@ -114,7 +112,10 @@ async fn hedging() -> anyhow::Result<()> {
     });
     tokio::time::sleep(std::time::Duration::from_secs(3)).await;
 
-    let mut payloads = load_fixture()?.payloads.into_iter();
+    let mut payloads = load_fixture("./tests/fixtures/hedging.json")
```

Comment:
I think we can delete the `hedging` test - `scenario_01` seems to capture everything - so its redundant.

Reply:
Ok, removing.